dnf.spec

```diff
@@ -1,7 +1,7 @@
 # default dependencies
 %global hawkey_version 0.52.0
 %global libcomps_version 0.1.8
-%global libmodulemd_version 1.4.0
+%global libmodulemd_version 2.9.4
```

Comment:
Can we use a lower version here? I think its not actually needed and fedora 32 for example still has `2.9.3`.

Reply:
Huh, 2.9.4 should have been released a long while ago; it looks like I forgot to file the Bodhi update...

We have 2.10.0 releasing this week, so it shouldn't be an issue. That said, I'll revise this to say 2.9.3 for now.